recipes/unidiff/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "unidiff" %}
+{% set version = "0.5.5" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/matiasb/python-{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 5d351e402191b10e6b5164e6ce40b683f5f63eaedccc9a17d3ed47bbe3873634
```

Comment:
We are better off pulling from the pypi source https://pypi.org/project/unidiff/.

Reply:
The pypi sources do not contain unit tests, which is probably a bug, but still it makes the pypi source not as useful for this package. I'll open an issue on the unidiff repo.

Reply:
Just made a PR: https://github.com/matiasb/python-unidiff/pull/61 PRs don't get merged quickly on this repo, so I'd rather not wait for it if you don't mind. Thanks for reviewing.

Reply:
It is ok for now. I don't consider running unit tests in the conda build needed. We are testing if a python package is installed properly here. An import does the trick. If the package does not work, that is an issue for the upstream package.